content/en/docs/Rleases/_index.md

```diff
@@ -1,3 +1,10 @@
+---
+title: Releases 
+linkTitle: "Releases"
+weight: 45
```

Comment:
The weight you have assigned places this between "Policies and Rules" and "Applying Policies". My suggestion is that it should go after "Troubleshooting" which means the weight should be greater than the weight assigned to that page.